laurat007/assignment3/Grid.java

```diff
@@ -0,0 +1,41 @@
+
+import java.util.TreeSet;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author Laura
+ *  Defining a grid with fixed number of columns and rows.
+ */
+class Grid {
+    
+    Dictionary dictionary;
+    int rows, columns;
+    char[][] grid;
+    
+    Grid(int r, int c, char[][] letterMatrix, Dictionary dict) {
+        dictionary = dict;
+        rows = r;
+        columns = c;
+        grid = new char[rows][columns];
```

Comment:
Is there any particular reason that you copy the whole char grid instead of just assigning it to letterMartix?

Reply:
I thought of building a new instance, but you have a point, it was not necessary.